### ISSUE TITLE
Add a List Control to the View

### ISSUE THREAD

**Issue by bask-alur-gh** [SAPDocuments/Tutorials#5001]
Tutorial URL: https://developers.sap.com/tutorials/sapui5-webide-add-list.html

Step1 - wondering why such a copy&paste method is being taught here when the WebIDE has a fantastic graphical layout editor is available that's easy & more intuitive?  Thanks for considering this.

**Comment by IObert** [comment]
This is a fair point. To be honest, the biggest reason why I didn't use the layout editor is that there is too much room for errors as it is hard to describe the needed actions in a very precise way. It is intuitive to drag and drop the controls when you know how to nest the controls. However, it is quite hard to describe the exact place where controls need to be dropped, eg to move the list item inside the list control and not next to it.
So I just went the easy way and preferred the text editor. I hope this makes sense.

I fully agree that the layout editor shouldn't be left out, so I added a box mentioning it in step 1.
